config/locales/clearance.fr.yml

```diff
@@ -0,0 +1,56 @@
+---
+fr:
+  clearance:
+    models:
+      clearance_mailer:
+        change_password: "Modifier votre mot de passe"
+  clearance_mailer:
+    change_password:
+      change_my_password: "Modifier mon mot de passe"
+      closing: "Si vous n'êtes pas à l'origine de cette demande, ignorez ce courriel. Votre mot de passe n'a pas été changé."
+      link_text: "Changer mon mot de passe"
+      opening: "Quelqu'un (nous espérons que c'est vous) a demandé que nous vous envoyions un lien pour changer votre mot de passe :"
+  flashes:
+    failure_after_create: "Mauvais courriel ou mot de passe."
+    failure_after_update: "Le mot de passe ne peut pas être vide."
+    failure_when_forbidden: "Veuillez vérifier l'URL ou essayer de soumettre
+      le formulaire à nouveau."
+    failure_when_not_signed_in: "Veuillez vous connecter pour continuer."
+  helpers:
+    label:
+      password:
+        email: Adresse courriel
+      password_reset:
+        password: Choisir un mot de passe
+    submit:
+      password:
+        submit: Réinitialiser le mot de passe
+      password_reset:
+        submit: Sauvegarder le mot de passe
+      session:
+        submit: Se connecter
+      user:
+        create: Créez votre compte
+  layouts:
+    application:
+      sign_in: Se connecter
+      sign_out: Se déconnecter
+  passwords:
+    create:
+      description: "Vous recevrez un courriel dans les prochaines minutes. Il contient des instructions pour changer votre mot de passe."
+    edit:
+      description: "Votre mot de passe a été réinitialisé. Choisissez un nouveau mot de passe ci-dessous."
+      title: Modifier votre mot de passe
+    new:
+      description: Pour recevoir un courriel contenant un lien pour de réinitialisation de votre mot de passe, veuillez entrer votre adresse courriel.
+      title: Réinitialiser votre mot de passe
+  sessions:
+    form:
+      forgot_password: "Mot de passe oublié ?"
```

Comment:
This is a nitpick, but I‘m wondering if in French it‘s common to have a space between the last word of the sentence and a punctuation mark like question mark (`?`) and colon (`:`)? I noticed this here and on line 12 but wasn‘t sure if this was intentional!

Reply:
French microtypography is quite the headache when you're considering internationalization (in addition to the fact that a French sentence is often 30 to 40% longer than an English sentence.). We have spaces before ":", "?", "!", ";" (ideally, non-breakable spaces `&nbsp;`), spaces between numbers and units "350&nbsp;€", special kind of spaces to separate numbers: "1&#8239;300&nbsp;€" (<= there's a `&#8239;` between the 1 and the 3).

I'm a huge nerd regarding french microtypo (and [develop things](https://rubygems.org/gems/jekyll-microtypo/versions/0.2.4) to automize typographic optimization whenever I can), so I tried to do things right (with the right spaces) in this translation. However, I quickly stopped, because it meant modifying too many templates to allow HTML characters in translation strings. So I only used "normal" spaces.

Reply:
Thanks for the explanation! I was curious, because I grew up in France but my French is super rusty (and it was also an era where we didn't type / use the computer as well, so I mostly wrote my papers by hand). Now I know 😅 
I'll let someone from the rest of the team give a 👍 to the thing overall before we can merge, but content-wise I think it's great! Thank you for doing this!

Reply:
Thanks **you and the dev team** for giving me the opportunity to help build something great. If you ever need a "French" support for when your own french is too rusty, feel free to contact me.

Reply:
I've done a quick translation review. Great job @borisschapira :-)